game/core/game_state.py

```diff
@@ -22,11 +36,14 @@ def __init__(self):  # type: ignore[no-untyped-def]
         self.tree_index = self.get_layer_index("interactables_blocking")
 
         # Player state
-        self.player_data = self.get_player_data()
-        self.center_x = self.player_data["x"]
-        self.center_y = self.player_data["y"]
-        self.inventory = self.load_inventory(self.player_data["inventory"])
-        self.item = self.load_item(self.player_data["item"])
+        try:
+            self.player_data = self.get_player_data()
+            self.set_player_data()
+        except Exception as e:
+            logger.warning(e)
+            logger.warning(f"Failed to load player state, using default data...")
```

Comment:
I would probably use `exception` if logging here:

```suggestion
        except Exception:
            logger.exception(f"Failed to load player state, using default data...")
```